package.json

```diff
@@ -2,7 +2,8 @@
   "name": "bayesjs",
   "version": "0.4.3",
   "description": "Inference on Bayesian Networks",
-  "main": "dist/bayes.js",
+  "main": "dist/index.js",
```

Comment:
The correct output file name is `bayes` 😄 

```
"main": "dist/bayes.js"
```

Reply:
I played around with the build process regarding our suggestions. But I just reached by using `webpack` always the situation that the type imports are missing. 
![1](https://user-images.githubusercontent.com/8091748/77418407-2252e980-6dc7-11ea-971f-80b90d470114.png)

This does not depend on `"main": "dist/bayes.js"`. I assume it has to do with the `webpack` build process itself, because the src/type folder is missing in the build result.

![2020-03-24 11_48_57-src](https://user-images.githubusercontent.com/8091748/77418491-4282a880-6dc7-11ea-9554-3bd35a5192d8.png)

Since I am not familiar with `webpack` I have currently no clue how to force the system to export the folder types as well. Do you have an suggestion for this issue?

Reply:
Hi, I finally found the error was because we are using `awesome-typescript-loader` instead `ts-loader` 🎉 
Also, I moved ` "test/**/*.ts` from `include` to `exclude`. That way this folder (test) will be not in the final result (in `dist`)

You can check my branch: [add-types](https://github.com/nolleto/bayesjs/tree/add-types)
or the commit: https://github.com/nolleto/bayesjs/commit/12ab0a133291de66eec29abddb09e52f6144b017
and check if this will work properly. 

If work, you can do the same changes in your PR 😄

Reply:
Very Nice, Thank you! 🎉 
I checked out your branch [add-types](https://github.com/nolleto/bayesjs/tree/add-types) built it, afterward my main project could import all types without any problems. 

I checked also the `dist`-folder it includes the types folder and all type definitions. I also run all tests locally and all of them worked find. 👍 

I think you should create a pull request from your branch, which would be much cleaner due to my past commit changing the `webpack` build process to `tsc`. I assume you will could close this pull request.